src/references.js

```diff
@@ -170,16 +170,16 @@ export default function findReferences(cell, globals) {
         identifier(node, parents);
         break;
       }
-      case "ArrayPattern":
+      case "ArrayPattern": {
+        for (const element of node.elements) {
+          checkConst(element, parents);
+        }
+        break;
+      }
       case "ObjectPattern": {
-        ancestor(
-          node,
-          {
-            Identifier: identifier,
```

Comment:
one minor nit: now that `identifier` is only used in one case, should we remove it and in-line the code? that'd remove the shadowing of `identifier` as well as remove the unnecessary mix of `parents` and `nodeParents` within that function.

Reply:
Oh, good catch, totally. Added! 👍